examples/tinywl-new/output.cpp

```diff
@@ -50,7 +53,13 @@ Output *Output::createPrimary(WOutput *output, QQmlEngine *engine, QObject *pare
 Output *Output::createCopy(WOutput *output, Output *proxy, QQmlEngine *engine, QObject *parent)
 {
     QQmlComponent delegate(engine, "Tinywl", "CopyOutput");
-    QObject *obj = delegate.create(engine->rootContext());
+
+    WOutputViewport *viewport = proxy->outputItem()->findChild<WOutputViewport*>({}, Qt::FindDirectChildrenOnly);
```

Comment:
Needs use `onscreenViewport` object, the outputviewport maybe changed after create copy output.

Reply:
done